crawler/src/main/java/com/movie/recommender/crawler/dataimport/MovieRepository.java

```diff
@@ -0,0 +1,6 @@
+package com.movie.recommender.crawler.dataimport;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import com.movie.recommender.common.model.movie.MovieDetails;
+
+public interface MovieRepository extends MongoRepository<MovieDetails, String> {
```

Comment:
move to `com.movie.recommender.crawler.repository`
service should be in `com.movie.recommender.crawler.service`